config/defaults.ini

```diff
@@ -202,13 +202,8 @@ kalite_content_user = uploader
 kalite_content_variant = endless
 kalite_content_version = 0.16
 
-# Preloaded Kolibri content. eos-kolibri-help is always selected by default.
+# Set to true to enable preloaded Kolibri content
 kolibri_content = false
-kolibri_content_app_version = 0.14.6
-kolibri_content_app_desktop_xdg_plugin_version = 1.0.8
-kolibri_content_channel_eos_help = e8a879742b2249a0a4b890f9903916f7
-kolibri_content_channel_endless_key = d29581bbf4394456971b8cb807219c39
-kolibri_content_extra_channels =
 
 # Kolibri automatic provisioning configuration
```

Comment:
Maybe we should also move these automatic provisioning settings to the `[kolibri]` section? (feel free to do this as a follow-up work)